x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -438,7 +439,7 @@ export class IndexTableUi extends Component {
             <Fragment>
               <EuiFlexItem>
                 <EuiSearchBar
-                  filters={this.getFilters()}
+                  filters={(this.getFilters().length > 0) ? this.getFilters() : null}
```

Comment:
@bmcconaghy I made this change to fix this issue when no filters were being passed.

![image](https://user-images.githubusercontent.com/324519/51283973-c14bac80-199e-11e9-892a-811dc293091e.png)